README.md

```diff
@@ -1,36 +1,78 @@
-
-[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7CHHJV)
+## FAQ
+<!-- REMEMBER TO CHANGE THE LINK ONCE THE PR IS READY !!!! -->
+- Remember to check the [Wiki](https://github.com/SpidFightFR/VulkanMod/wiki) we wrote before asking for support!
 
 ## Installation
 
-### Download
-[![CurseForge](https://cf.way2muchnoise.eu/title/635429_Get_Mod.svg?badge_style=flat)](https://www.curseforge.com/minecraft/mc-mods/vulkanmod)
-
-[![CurseForge](https://cf.way2muchnoise.eu/full_635429_downloads.svg?badge_style=flat)](https://www.curseforge.com/minecraft/mc-mods/vulkanmod)
-
-- Fabric mod loader is needed.
-- Move `VulkanMod.jar` to `mods` folder of your Minecraft Instance.
-
-## Known Issues
+### Download Links:
+
+- [![CurseForge](https://cf.way2muchnoise.eu/full_635429_downloads.svg?badge_style=flat)](https://www.curseforge.com/minecraft/mc-mods/vulkanmod)
+
+- [![Modrinth Downloads](https://img.shields.io/modrinth/dt/JYQhtZtO?logo=modrinth&label=Modrinth%20Downloads)](https://modrinth.com/mod/vulkanmod/versions)
+
+- [![GitHub Downloads (all assets, all releases)](https://img.shields.io/github/downloads/xCollateral/VulkanMod/total?style=flat-square&logo=github&label=Github%20Downloads)](https://github.com/xCollateral/VulkanMod/releases)
+
+### Install guide:
+>1) Install the [fabric modloader](https://fabricmc.net).
+>1) Download and put the `Vulkanmod.jar` file into `.minecraft/mods`
+>1) Enjoy !
+
+## Useful links
+<table>
+    <tr>
+      <th> Discord server</th>
+      <th> Ko-Fi</th>
+    </tr>
+  <tr>
+    <td style="text-align:center"> 
+        <a href="https://discord.gg/FVXg7AYR2Q"> 
+            <img alt="Discord" align="top" src="https://img.shields.io/discord/963180553547419670?style=flat-square&logo=discord&logoColor=%23FFFFFF&label=Vulkanmod%20official%20discord%20server&labelColor=%235865F2&color=%235865F2">
+        </a>
+     </td>
+    <td>
+        <a href="https://ko-fi.com/V7V7CHHJV">
+            <img alt="Static Badge" align="top" src="https://img.shields.io/badge/KoFi-%23ff5e5b?logo=ko-fi&logoColor=%23FFFFFF&link=https%3A%2F%2Fko-fi.com%2FV7V7CHHJV">
+        </a>
+    </td>
+  </tr>
+</table>
+
+
+## Features
+
+### Optimizations:
+>- [x] GUI optimizations
+>- [x] Multiple chunk culling algorithms
+>- [x] Reduced CPU overhead
+>- [x] Improved GPU performance
+>- [x] Indirect Draw mode (reduces Vram usage - optional)
+>- [x] Chunk rendering optimizations
+
+### New content:
+>- [x] Native Wayland support
+>- [x] GPU selector
+>- [x] Windowed fullscreen mode
+>- [x] Revamped graphic settings menu
+>- [x] Risizable render frame queue
+>- [ ] Shader support
+>- [ ] Removed Herobrine
 
-- Linux and macOS may not work.
 
 ## Notes
-This is still a pre-alpha mod. Bugs are to be expected.
+- This mod is still in developpment, please report issues in the [issue tab](https://github.com/xCollateral/VulkanMod/issues) with logs attached!
```

Comment:
Just letting you know that there is a 'p' to much in 'development'.

Reply:
Thanks for the heads up !